cli/git.go

```diff
@@ -13,6 +13,10 @@ import (
 	cfg "github.com/flynn/flynn/cli/config"
 )
 
+func inGitRepo() bool {
+	return exec.Command("git", "rev-parse", "--git-dir").Run() == nil
```

Comment:
Should we memoize this? If we don't we'll exec multiple times in a single command run.